test/surface/run-solid-test-suite.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Slight preference for putting this in a `system` folder over `surface` to match the existing testing plan. @joachimvh?

Reply:
I was thinking that it might make most sense to put it in the `.travis` folder next to the other script we already have? That way the tsconfig doesn't need to be edited either.

Reply:
I'd rather move that other Travis test to the same folder (there was some discussion about that when it was created).

Reply:
That works as well, would just make sure they're not spread out all over the repository 😄

Reply:
I put it I `test/system/` for now.